runtime/src/assets/stylesheets/planner-overrides.less

```diff
@@ -1,5 +1,5 @@
-// _layout.scss overrides
+// _layout.less overrides
 
-.btn {
+/*.btn {
```

Comment:
You can just leave this file blank. If you need this for reference, please add a comment stating as such.